src/lib/common/Kconfig

```diff
@@ -121,6 +121,18 @@ config PLATFORM_RIOTOS
             Soletta will do its best to offer the same experience on
             all platforms, including RIOt.
 
+config PLATFORM_CONTIKI
+	bool "contiki"
```

Comment:
I think you should check for some contiki header in dependencies.json and depend on it here, otherwise contiki platform will show as a build option even when it is not.